modules/juce_audio_plugin_client/AU/juce_AU_Wrapper.mm

```diff
@@ -1225,16 +1225,26 @@ void audioProcessorParameterChangeGestureEnd (AudioProcessor*, int index) overri
         sendAUEvent (kAudioUnitEvent_EndParameterChangeGesture, index);
     }
 
-    void audioProcessorChanged (AudioProcessor*) override
+    void audioProcessorChanged (AudioProcessor*, int flags) override
     {
-        PropertyChanged (kAudioUnitProperty_Latency,       kAudioUnitScope_Global, 0);
-        PropertyChanged (kAudioUnitProperty_ParameterList, kAudioUnitScope_Global, 0);
-        PropertyChanged (kAudioUnitProperty_ParameterInfo, kAudioUnitScope_Global, 0);
+        if (flags & AudioProcessorListener::Flags::latencyUpdate)
+            PropertyChanged (kAudioUnitProperty_Latency, kAudioUnitScope_Global,
+                             0);

```

Comment:
This looks like our clang-format, but is it what JUCE does as well? I would expect them to want to leave the line intact

Reply:
Thanks. I'll go with the JUCE way. this one slipped my interactive committing post formatting.